src/server.cpp

```diff
@@ -31,8 +36,13 @@ struct fmt::formatter<T, std::enable_if_t<std::is_base_of<py::object, T>::value,
 	}
 };
 
-using json_loads_t = std::function<py::object(std::string)>;
-using json_dumps_t = std::function<std::string(py::object)>;
+struct Codec{
```

Comment:
```
/home/runner/work/tuber/tuber/src/server.cpp:39:8: warning: ‘Codec’ declared with greater visibility than the type of its field ‘Codec::loads’ [-Wattributes]
   39 | struct Codec{
      |        ^~~~~
/home/runner/work/tuber/tuber/src/server.cpp:39:8: warning: ‘Codec’ declared with greater visibility than the type of its field ‘Codec::dumps’ [-Wattributes]
[ 50%] Linking CXX executable tuberd
```

Reply:
This one is a little peculiar to me; looking this up on SO suggests that slapping `DLL_LOCAL` on the class would probably quiet it down, although I'm a bit surprised that it's a concern here, as I thought we were building an excutable rather than a library and wouldn't care much about visibility? Doing most of my development with clang producing Mach-O binaries may have also caused me not to notice this. At any rate, I'm happy to add the attribute if that's the right fix.

Reply:
I didn't suggest a fix because I find this stuff baffling. I don't feel strongly about any of it.